cpp-package/example/alexnet.cpp

```diff
@@ -228,6 +228,7 @@ int main(int argc, char const *argv[]) {
   }
 #endif
 
+  TRY
```

Comment:
can we just add exception handling to cpp-package the way it is done in python. Every C API call is inside a check_call function which checks the return code and throws corresponding frontend exception.

Reply:
It is certainly possible to add exception handling in cpp-package API itself.  However, it will require careful design and we may require to change the cpp-package API as well. 
We will certainly consider this enhancement as a part of C++ API enhancement project.